.travis.yml

```diff
@@ -28,7 +28,7 @@ before_install:
   - source $HOME/google-cloud-sdk/path.bash.inc
   - gcloud config set project broad-jade-dd
   - ./render-configs.sh $TRAVIS_TOKEN
-  - gcloud auth activate-service-account --key-file /tmp/jade-dev-account.json
+#  - gcloud auth activate-service-account --key-file /tmp/jade-dev-account.json
```

Comment:
Can you either add a comment above this about why this is commented out or remove this line?

Reply:
Yes, done.